### DISCUSSION TITLE
Help with UC on chromium 116

### DISCUSSION THREAD

**Discussion by SoplicaSoftware** [ultrafunkamsterdam/undetected-chromedriver#1492] [Q&A]
Hi, Im running a little project on ubuntu server and Im having issues with UC after chromium updated to 116. Does anyone know when will the author fix this issue? I was concidering using one of the unofficial fixes, but this issue seems to be present for a few days, so maybe it will be fixed soon. Also does anybody know if theres a reliable way to download older versions (preferably newest version that UC works properly on) of chromium on platform like ubuntu? I would be thankful for any answer.

**Comment by NawazSellerx** [comment]
In Mac also chrome is updated to 116 version.

Message: unknown error: cannot connect to chrome at XXX.X.X.X:XXXXX
from session not created: This version of ChromeDriver only supports Chrome version 114
Current browser version is 116.0.5845.96

**Comment by gitvishal96** [comment]
run command in the terminal (like vs code ) -`pip install --upgrade undetected_chromedriver`

**Comment by ortizjeison** [comment]
I just updated chrome to the latest stable version, now it's 117 and UC is not working as expected (Is not being undetectable) does anyone knows if there is any upcoming updates to fix it?

**Reply by oculunto** [reply]
There is a new version announced for weeks now: https://github.com/ultrafunkamsterdam/undetected-chromedriver/discussions/1420
In the meantime you can use seleniumBase: https://github.com/seleniumbase/SeleniumBase

**Reply by w956205480** [reply]
**Sorry, but I upgraded to 3.5.2 with "pip install --upgrade undetected_chromedriver" and it still doesn't work. How do I upgrade to this new version #1420?**

**Reply by oculunto** [reply]
It is not out yet sadly, try to adapt your code with SeleniumBase meanwhile
